chaosazure/machine/actions.py

```diff
@@ -10,7 +11,10 @@
 from chaosazure.machine.constants import RES_TYPE_VM
 from chaosazure.rgraph.resource_graph import fetch_resources
 
-__all__ = ["delete_machine", "stop_machine", "restart_machine"]
+__all__ = ["delete_machine", "stop_machine", "restart_machine",
+           "start_machines"]
+
+stopped_machines = []
```

Comment:
Could we protect this variable with a lock? Either make it a [queue](https://docs.python.org/3/library/queue.html#queue.Queue) or add a bunch of internal functions to protect its access.

Reply:
We removed the state